mots.csl

```diff
@@ -0,0 +1,299 @@
+                <label form="short"/>
+              </names>
+            </if>
+          </choose>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="auteurs-ouvrage">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <text term="in" suffix=" "/>
+        <group delimiter=", ">
+          <choose>
+            <if variable="author">
+              <names variable="editor" delimiter=", ">
+                <name sort-separator=" " and="text" delimiter=", " initialize-with=". " delimiter-precedes-last="never"/>
+                <label form="short" prefix=" (" suffix=")"/>
+              </names>
+            </if>
+          </choose>
+          <choose>
+            <if variable="author editor" match="any">
+              <names variable="translator" delimiter=", ">
+                <name sort-separator=" " and="text" delimiter=", " initialize-with=". " delimiter-precedes-last="never"/>
+                <label form="short" prefix=" (" suffix=")"/>
+              </names>
+            </if>
+          </choose>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="titre-ouvrage">
+    <text variable="container-title" form="short" font-style="italic"/>
+    <choose>
+      <if type="article-journal">
+        <text variable="volume" prefix=", n° " suffix=","/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="lieu-edition">
+    <choose>
+      <if type="book chapter thesis report paper-conference" match="any">
+        <group delimiter=", ">
+          <text variable="publisher-place" form="short"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="editeur">
+    <choose>
+      <if type="book chapter thesis report paper-conference" match="any">
+        <group delimiter=", ">
+          <text variable="publisher" form="short"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="publication">
+    <choose>
+      <if type="thesis">
+        <group>
+          <text value="Thèse de doctorat soutenue à "/>
+          <text macro="lieu-edition"/>
+          <text macro="date-publication" prefix=", "/>
+        </group>
+      </if>
+      <else>
+        <group delimiter=", ">
+          <text macro="lieu-edition"/>
+          <text macro="editeur"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="pages-chapitre">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <text variable="volume" suffix=" :"/>
+        <text variable="page" prefix="p. "/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="pages-article">
+    <choose>
+      <if type="article-journal">
+        <text variable="page" prefix="p. "/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="type-acces">
+    <group delimiter=" ">
+      <text variable="URL" prefix=" "/>
+      <choose>
+        <if variable="URL">
+          <group prefix="(consulté le " suffix=")">
```

Comment:
it's much preferred to avoid hardcoding strings and using (and if necessary re-defining) terms instead.